docs/pages/using-nhs-notify/personalisation.md

```diff
@@ -67,17 +57,14 @@ For example, if you wanted to include 'practice' as a personalisation field, the
 
 Read the [API documentation](https://digital.nhs.uk/developer/api-catalogue/nhs-notify#post-/v1/message-batches) to find out where to put this in your request.
 
-### If you're using the Message Exchange for Social Care and Health (MESH)
+### If you're using NHS Notify MESH
 
 Include the personalisation fields as columns in your CSV file.
 
-The column names should start with 'personalisation_', followed by the same wording as the placeholders in your template.
+The column names should start with 'personalisation\_', followed by the same wording as the placeholders in your template.
 
 For example, if you wanted to include 'practice' as a personalisation field, the column name would be:
 
 {% include components/inset-text.html
-    text='nhsNumber,requestItemRefId,dateOfBirth,personalisation_practice'
+    text='nhsNumber,requestItemRefId,personalisation_practice'
```

Comment:
Should this also be formatted as code? I.e:
`text='<code>nhsNumber,requestItemRefId,personalisation_practice</code>'`

Reply:
Not sure on this, they're column headers for a CSV file so not sure if that's 'code' or just data?